whipper/common/mbngs.py

```diff
@@ -281,9 +281,9 @@ def musicbrainz(discid, country=None, record=False):
 
     Example disc id: Mj48G109whzEmAbPBoGvd4KyCS4-
 
-    @type  discid: str
+    :type  discid: str
 
-    @rtype: list of L{DiscMetadata}
+    :rtype: list of :any:`DiscMetadata`
```

Comment:
If we’re changing `list/dict of SomeThing` to `list/dict(SomeThing)`, then probably this should also be changed?